file-formats/chkv/type/zif_atc_aff_chkv_v1.intf.abap

```diff
@@ -0,0 +1,147 @@
+      "! <p class="shorttext">Option</p>
+      "! Sign option of the range
+      option TYPE ty_option,
+      "! <p class="shorttext">Low</p>
+      "! The low value of the range
+      low    TYPE string,
+      "! <p class="shorttext">High</p>
+      "! The high value of the range
+      high   TYPE string,
+    END OF ty_range,
+
+    ty_ranges      TYPE STANDARD TABLE OF ty_range WITH DEFAULT KEY,
+
+    "! <p class="shorttext">Entry in Value List</p>
+    "! A parameter value in the parameter value list
+    ty_param_value TYPE string,
+
+    "! <p class="shorttext">Parameter</p>
+    "! A parameter of ATC check
+    BEGIN OF ty_properties,
+      "! <p class="shorttext">Parameter Name</p>
+      "! The parameter name
+      "! $required
+      name             TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Parameter Type</p>
+      "! The parameter type
+      "! $required
+      type             TYPE string,
+      "! <p class="shorttext">Parameter Value</p>
+      "! The value of the parameter
+      value            TYPE string,
+      "! <p class="shorttext">List of Parameter Values</p>
+      "! List of values for a multi-value check parameter
+      value_list       TYPE STANDARD TABLE OF ty_param_value WITH DEFAULT KEY,
+      "! <p class="shorttext">List of Range Values</p>
+      "! List of range values for a check parameter
+      value_range_list TYPE ty_ranges,
+    END OF ty_properties,
+
+    "! <p class="shorttext">Parameters</p>
+    "! List of check parameters
+    ty_properties_tab TYPE STANDARD TABLE OF ty_properties WITH DEFAULT KEY.
+
+  TYPES:
+    "! <p class="shorttext">Check</p>
+    "! Check with its parameters
+    BEGIN OF ty_check_parameter_json,
+      "! <p class="shorttext">Check Name</p>
+      "! The name of check
+      check_name TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Check Parameters</p>
+      "! Parameters of check
+      parameters TYPE ty_properties_tab,
+    END OF ty_check_parameter_json.
+
+  TYPES:
+      "! <p class="shorttext">Selected Checks</p>
+      "! List of selected checks of the ATC check variant
+        ty_selected_checks TYPE STANDARD TABLE OF ty_check_parameter_json WITH DEFAULT KEY.
+
+
+  TYPES:
+    "! <p class="shorttext">ATC Check Variant Properties</p>
+    "! ATC check variant properties
+    BEGIN OF ty_main,
+      "! <p class="shorttext">Schema</p>
+      "! Format version
+      "! $required
+      schema             TYPE string,
+      "! <p class="shorttext">Header</p>
+      "! Header
+      "! $required
+      header             TYPE zif_aff_types_v1=>ty_header_60,
+      "! <p class="shorttext">Remote-Enabled Flag</p>
+      "! The ATC check variant is remote-enabled
+      remote_enabled     TYPE abap_bool,
+      "! <p class="shorttext">Referenced SCI Variant</p>
+      "! Name of the referenced ATC check variant
+      referenced_variant TYPE zif_aff_types_v1=>ty_object_name_30,
```

Comment:
Is the referenced variant supported by CHKV?

Reply:
currently not but we have already provided this in the abap type, so we can support it later in the editor

Reply:
If we do not support it, I would remove it.